conda-forge.yml

```diff
@@ -1,3 +1,6 @@
+build_platform:
+  linux_aarch64: linux_64
+  osx_arm64: osx_64
 provider:
   linux_aarch64: default
```

Comment:
Since this PR is limited to cross-compiling for osx-arm64, please remove all references to the linux-aarch64 build in `conda-forge.yml` and then rerender the feedstock

Reply:
Please have a look now